testing/trino-product-tests/src/main/java/io/trino/tests/utils/QueryExecutors.java

```diff
@@ -27,19 +27,37 @@
 {
```

Comment:
So how previously this retry was not working at all, right?

Reply:
Yes, retries didn't work in `onHive()`